src/app/sections/hero-section.tsx

```diff
@@ -1,73 +1,71 @@
 import React from "react";
 import homeData from "@/content/home.json";
 import Image from "next/image";
-import NavBar from "../components/NavBar";
+import NavBar from "../components/nav-bar";
+
 export default function HeroSection() {
   const { hero } = homeData;
   return (
-    <div className="flex flex-col bg-gradient-to-r from-[#9ec5f7] to-darkblue">
+    <div className="bg-gradient-to-r from-[#9ec5f7] to-darkblue">
       <NavBar />
-      <div className="lg:max-w-7xl lg:flex-row md:justify-between gap-4 py-10 flex flex-col">
-        <div className="lg:w-full lg:pl-20 lg:p-0 md:p-10 p-3 text-center">
+      <div className="flex flex-col gap-4 py-10 md:justify-between lg:flex-row lg:max-w-7xl lg:pb-5">
```

Comment:
center the content. Else it would be left aligned on wide screens

```suggestion
      <div className="mx-auto flex flex-col gap-4 py-10 md:justify-between lg:flex-row lg:max-w-7xl lg:pb-5">
```